lib/screens/customer_product_detail_page.dart

```diff
@@ -147,7 +152,7 @@ class _CustomerProductDetailPageState extends State<CustomerProductDetailPage> {
         return DropdownMenuItem<ItemColor>(
           value: value,
           child: _buildDropdownMenuItem(
-            "色：",
+            "",
```

Comment:
This parameter is entirely not necessary. Can you remove this parameter in the `_buildDropdownMenuItem` function and simplify the code?